recipes/gtest/all/conanfile.py

```diff
@@ -3,9 +3,10 @@
 from conan import ConanFile
 from conan.errors import ConanInvalidConfiguration
 from conan.tools.build import check_min_cppstd
-from conan.tools.files import copy, get, replace_in_file, rm, rmdir
+from conan.tools.files import apply_conandata_patches, copy, get, replace_in_file, rm, rmdir
 from conan.tools.cmake import CMake, cmake_layout, CMakeToolchain
 from conan.tools.microsoft import is_msvc, is_msvc_static_runtime
+from conan.tools.scm import Version
 
 required_conan_version = ">=1.51.1"
```

Comment:
```suggestion
required_conan_version = ">=1.52.0"
```

export_conandata_patches requires 1.52.0